include/swift/Runtime/Config.h

```diff
@@ -216,6 +216,7 @@
 // But some of the runtime entries are invoked directly from
 // the foundation. Therefore they should be visible.
 #define SWIFT_RT_ENTRY_VISIBILITY SWIFT_RUNTIME_EXPORT
+#define SWIFT_EXTERN_RT_ENTRY_VISIBILITY SWIFT_EXTERN_RUNTIME_EXPORT
```

Comment:
There are two such situations in Casting.cpp
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Casting.cpp#L266
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Casting.cpp#L2423

As well as in Enum.cpp

- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Enum.cpp#L203
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Enum.cpp#L2

As well as Heap.cpp

- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Heap.cpp#L25
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Heap.cpp#L34

As well as HeapObject.cpp
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Heap.cpp#L34
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L307
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L330
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L386
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L402
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L410
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L415
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L432
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L463
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L475
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/HeapObject.cpp#L605

As well as Metadata.cpp
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Metadata.cpp#L230
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/Metadata.cpp#L2364

As well as SwiftObject.mm
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/SwiftObject.mm#L1345
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/SwiftObject.mm#L1444
- https://github.com/apple/swift/blob/master/stdlib/public/runtime/SwiftObject.mm#L1455